nixos/maintainers/scripts/ec2/amazon-base-config.nix

```diff
@@ -1,5 +1,5 @@
 { modulesPath, ...}:
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-config.nix" ];
+  imports = [ ../../../../nixos/modules/virtualisation/amazon-image.nix ];
```

Comment:
@nbp @shlevy is this the right way to be excising `modulesPath`? I've seen people do it with `<nixpkgs>` but that doesn't seem ideal.